custom/icds_reports/utils/aggregation_helpers/distributed/agg_awc.py

```diff
@@ -228,7 +227,8 @@ def updates(self):
             ) as cases_person_adolescent_girls_11_14_all,
             sum(
                 CASE WHEN %(month_end_11yr)s > dob AND %(month_start_14yr)s <= dob AND sex = 'F'
-                    AND migration_status IS DISTINCT FROM 1
+                    AND ((agg_migration.is_migrated=1 AND
+                    agg_migration.migration_date < %(start_date)s)::integer IS DISTINCT FROM 1)
```

Comment:
Looks like this one wasn't updated but the rest are good.

Reply:
Done!